job_templates/ci_job.xml.em

```diff
@@ -133,6 +133,8 @@ if [ "$CI_USE_OPENSPLICE" = "true" ]; then
 fi
 if [ -n "${CI_ROS2_REPOS_URL+x}" ]; then
   export CI_ARGS="$CI_ARGS --repo-file-url $CI_ROS2_REPOS_URL"
+else
+  export CI_ARGS="$CI_ARGS --repo-file-url @default_repos_url"
```

Comment:
Since both blocks contain similar code I would suggest to deduplicate the identical part. If `CI_ROS2_REPOS_URL` is empty initialize it with `@default_repos_url`. Then always append `--repo-file-url` to `CI_ARGS`.

```
```